sunkit_image/tests/test_stara.py

```diff
@@ -0,0 +1,28 @@
+import astropy.units as u
+import numpy as np
+
+from sunkit_image.stara import stara
+from sunkit_image.tests.helpers import figure_test
+
+
+def test_stara(hmi_map):
+    hmi_upscaled = hmi_map.resample((1024, 1024) * u.pixel)
+    result = stara(hmi_upscaled)
+    assert isinstance(result, np.ndarray)
+    assert result.shape == hmi_upscaled.data.shape
+    total_true_value_count = np.array(result)
+    assert total_true_value_count.sum() == 5516
```

Comment:
This is weird since when I run this on my local machine this test passes. 
This test is actually counting the total number of True values in the 2D array `result` .